_pages/install/integrations/opf-integration/opf-child-pages/1-open-payment-framework-in-spartacus.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Open Payment Framework in Spartacus
+---
+
+Open payment framework allows you to quickly integrate your preferred digital payment service providers in Spartacus, and removes the need to code, integrate, and deploy extensions to the SAP Commerce Cloud codebase.
+
+Open payment framework requires SAP Commerce Cloud 2211.32 or newer. Before integrating open payment framework with Spartacus, you need to configure open payment framework in SAP Commerce Cloud. For more information, see [Open Payment Framework](link to backend OPF docs).
+
+## Enabling Open Payment Framework in Spartacus
+
+To enable open payment framework, install the `@spartacus/opf` integration library. For more information, see [Installing Additional Composable Storefront Libraries](link).
```

Comment:
There should be two use cases if I understand correctly:
1. Enable OPF for a brand new Composable Storefront.
2. Enable OPF for an existing Composable Storefront.

Reply:
Hello @opf-postman 👋 🙂 

Thank you for this feedback! If you could follow up with me on Slack, that would be great! 🙏 🙂